paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/source/CompactorSourceBuilder.java

```diff
@@ -96,7 +95,7 @@ public CompactorSourceBuilder withPartitionIdleTime(@Nullable Duration partition
         } else {
             compactBucketsTable = compactBucketsTable.copy(batchCompactOptions());
             ReadBuilder readBuilder =
-                    compactBucketsTable.newReadBuilder().withFilter(partitionPredicate);
+                    compactBucketsTable.newReadBuilder().withFilter(partitionPredicate).dropStats();
```

Comment:
If (MANIFEST_DELETE_FILE_DROP_STATS is true) drop stats? Also work for `isContinuous`.

Reply:
Thx, done